integration/xgboost/encryption_plugins/v.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python
```

Comment:
Add license header

Reply:
for this testing script, maybe we move it to the example folder?

Reply:
yes, I should remove it